websocket.go

```diff
@@ -27,9 +36,21 @@ var WsCodec = &manet.NetCodec{
 	ConvertMultiaddr: ConvertWebsocketMultiaddrToNetAddr,
 	ParseNetAddr:     ParseWebsocketNetAddr,
 }
+var WssCodec = &manet.NetCodec{
+	NetAddrNetworks:  []string{"websocket secure"},
```

Comment:
Is `websocket secure` okay for the network name? The other PR was using `websocket+tls`.

Reply:
WebSocket Secure is the official name, so it LGTM.